spec/models/maths_and_physics/school_eligibility_spec.rb

```diff
@@ -64,6 +64,16 @@
         end
       end
 
+      context "and it is a alternative provision school" do
+        let(:school_attributes) { {school_type: :pupil_referral_unit, phase: :not_applicable} }
+        it { is_expected.to be false }
+
+        context "and it has a statutory high age of over 11" do
+          let(:school_attributes) { {statutory_high_age: 16} }
```

Comment:
The `school_attributes` here don't reflect the nested context we're in. I would expect them to contain a `school_type` belonging to an alternative provision school, at the moment the `school_type` will be the one from the factory.